widgetssdk/src/main/java/com/glia/widgets/core/visitor/VisitorMediaStateListener.java

```diff
@@ -0,0 +1,9 @@
+package com.glia.widgets.core.visitor;
+
+import com.glia.androidsdk.comms.VisitorMediaState;
+
+public interface VisitorMediaStateListener {
+    void onNewVisitorMediaState(VisitorMediaState visitorMediaState);
+
+    void onHoldChanged(boolean isOnHold);
```

Comment:
could onHold be part of VisitorMediaState?

I would follow same naming here, whether: `on...Changed` or `onNew...`